Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+- Open-source hardware: fpnew (ETH Zürich), first verified in CV32E40Pv2
+- Eclipse Foundation, GitHub
+- Digilent Genesys 2 board
+- Linter: verible
+- Simulators: Verilator (open-source), Siemens Questa, Synopsys VCS, Cadence Xcelium
+- Synthesis: Vivado (AMD, ex Xilinx), Libero (Microchip), Quartus (Intel, ex Altera), Synopsys Design Compiler (ASIC)
+
+## List of project outputs
+
+### Documentation:
+
+The CVA6 document structure is:
+- Requirement specification
+    - Identifies features agreed upon
+    - “What” defined as requirements with identifiers
+    - Can be seen as a datasheet
+- User Manual
+    - Includes more details than the requirement specification
+    - For CVA6 integrators and users: HW, SW, ASIC, FPGA… viewpoints
+- Design document
+    - Explains the “How”: design choices…
+    - Useful for verification, maintenance, next projects and certain certification schemes
+- Design Verification Plans
+    - DVplan, Verification Plan, Vplan: same meaning
+    - Feature-by-feature listing of the Device Under Test
+        - and a description of how it will be verified
+        - and how we know when it is verified (coverage).
+    -   Prepared with the [VPTOOL](https://github.com/openhwgroup/core-v-verif/tree/master/tools/vptool) utility
+
+The user manual and design document are inputs for design verification. They are joint documents for all configurations.
+Thanks to templating, it will be possible to generate versions specific to some given configurations.
+
+DVplans can be commonalized between some configurations thanks to the support of configuration fields by VPTOOL.
+
+### Design 
+
+- CVA6 configurable RTL source code (TRL5 target)
+- Subsystems and FPGA designs, a.k.a. APU, to exercise CVA6 on development boards...
+
+All CVA6 configurations share the same source code and are differentiated thanks to SystemVerilog parameters.
+
+### Verification:
+
+- Versatile generic testbench with adaptation layers for CVA6
+    - Subset compliant with _sustainable open-source solution_ expectactions
+- Test sequences
+- Verification results
+     - Including code coverage and functional coverage
+
+The sub-projects will provide more details. They will also identify what then can reuse between different configurations.
+
+### Software:
+
+- Baremetal BSP for Genesys 2 FPGA board featuring CVA6
+- Linux ports for CV32A6 and CV64A6 on the Genesys 2 board, based on UBoot bootloader, OpenSBI firmware and Yocto embedded Linux distribution builder
+- Toolchains (parameters for GCC compilation...)
+
+FreeRTOS is a product of a sister project, that has been kept in sync with the CVA6 project.
+
+### Release process
+
+The OpenHW Group has a release process adapted to single configurations (CV32E40P...)
+Some additions to this process are needed to address a project and joint repository that will generate several TRL5 cores.
+
```

Comment:
Let's get a work stream going to address exactly the release process. Just a comment :)

Reply:
That's the first line in the list of activities (line ~422).
I have no experience with open-source releases. Support will be appreciated.